data/BGC0001444.json

```diff
@@ -19,11 +19,21 @@
                 "AAAAAAAAAAAAAAAAAAAAAAAA"
             ],
             "version": "2.0"
+        },
+        {
+            "comments": [
+                "Merged annotations from duplicate entry BGC0001538"
+            ],
+            "contributors": [
+                "SJG3CBEHSM6A3I74YXLH6GLA"
+            ],
+            "version": "2.1"
         }
     ],
     "cluster": {
         "biosyn_class": [
-            "Other"
```

Comment:
No, I think Other, with a subclass of "shikimate derived" would be correct

Reply:
Reverted that bit, added the subclass